Dependencies/xr/Include/XR.h

```diff
@@ -149,6 +168,9 @@ namespace xr
                 ~Frame();
 
                 void GetHitTestResults(std::vector<HitResult>&, Ray) const;
+                void UpdatePlanes(std::map<NativePlaneIdentifier, Plane*>& existingPlanes, std::vector<Plane>& newPlanes, std::vector<Plane*>& removedPlanes) const;
```

Comment:
`std::vector<Plane>` instead of `Plane*`?

Reply:
I am using a plane pointer so that we can edit the plane in place.  I don't think you can do std::vector<Plane&> right?

Reply:
Yes, as long as the vector components are non-const, you can edit them.

Reply:
This comment is outdated now.